app/src/main/res/values/dimens.xml

```diff
@@ -23,4 +23,8 @@
     <dimen name="dp_2">2dp</dimen>
     <dimen name="sp_18">18sp</dimen>
     <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_90">90dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_90` appears to be unused
 Unused resources make applications larger and slow down builds.